src/main/clojure/jarl/parser.clj

```diff
@@ -243,8 +243,23 @@
     (fn [state]
       (eval/eval-blocks blocks state))))
 
+(defn- stringify-coll-keys [val]
```

Comment:
I'm certain there is a more concise way of solving this 😅

Reply:
I was looking into `prewalk`/`postwalk`, but I couldn't figure out a concise way of knowing if what was being modified was a map key or -value.